poetry.lock

```diff
@@ -0,0 +1,540 @@
+[[package]]
```

Comment:
https://python-poetry.org/docs/basic-usage/#commit-your-poetrylock-file-to-version-control